src/Datadog.Trace.ClrProfiler.Managed/Integrations/AspNetIntegration.cs

```diff
@@ -19,9 +19,9 @@ public static class AspNetIntegration
         ///     Calls the underlying Init() For an HttpApplication and traces the request.
         /// </summary>
         /// <param name="thisObj">The HttpApplication instance ref.</param>
-        [InterceptMethod(
-            TargetAssembly = "System.Web",
-            TargetType = "System.Web.HttpApplication")]
+        // [InterceptMethod(
```

Comment:
Why not just delete instead of comment out? Git history, can always get this back.